impl/java/test/src/test/kotlin/br/com/guiabolso/events/test/EventHandlerMatchersTests.kt

```diff
@@ -0,0 +1,60 @@
+package br.com.guiabolso.events.test
+
+import br.com.guiabolso.events.builder.EventBuilder
+import br.com.guiabolso.events.model.RequestEvent
+import br.com.guiabolso.events.model.ResponseEvent
+import br.com.guiabolso.events.server.handler.EventHandler
+import io.kotest.assertions.throwables.shouldThrowAny
+import io.kotest.core.spec.style.FunSpec
+
+class EventHandlerMatchersTests : FunSpec({
+    
+    test("should reject") {
+        shouldThrowAny { SuccessfulValidateEventHandler.shouldReject(requestEvent(42)) }
+        shouldThrowAny { FailingValidateEventHandler.shouldReject(requestEvent(42)) }
+        
+        FailingValidateEventHandler.shouldReject<IllegalArgumentException>(requestEvent(41))
+    }
+    
+    test("should not reject") {
```

Comment:
sei la, eu teria separado pelos casos e não só pelo `reject`, `not reject`
nao ficou claro pra mim que 42 da erro e 41 passa
fiquei quebrando a cabeça e depois que desci